app/utilities/Information.java

```diff
@@ -0,0 +1,20 @@
+package utilities;
+
+/**
+ * Created by Ejub on 8.3.2016.
+ */
+public class Information {
+    private String information;
+
+    public Information(String information) {
+        this.information = information;
+    }
+
+    public String getInformation() {
+        return information;
+    }
+
+    public void setInformation(String information) {
+        this.information = information;
+    }
+}
```

Comment:
This is something like success message. You should add it to all similar methods (i.e. creation and update methods), or you should leave body empty. Leave it empty for now.